### DISCUSSION TITLE
Please add the abaility to use a custom login server

### DISCUSSION THREAD

**Discussion by TimDowker** [hassio-addons/app-tailscale#173] [Ideas]
Pretty much as the title requests - pleaase add the configuration option to use a custom login server.  I know this is an advanced feature so if you need to hide it under the "Show unused optional configuration options", that would be ok.

Thank you

**Comment by DennisGaida** [comment]
Did you check the documentation, particularly the `login_server` option? https://github.com/hassio-addons/addon-tailscale/blob/main/tailscale/DOCS.md

**Comment by lmagyar** [comment]
FYI: This was added at 2023.04.14, a week after this question was asked.
